bin/setup

```diff
@@ -2,7 +2,6 @@
 require "fileutils"
 
 APP_ROOT = File.expand_path("..", __dir__)
-APP_NAME = "suri"
```

Comment:
I've been leaving this line in... maybe not necessary as long as tests pass.

Reply:
i've been letting it vanish -- i'm guessing it's dandling and not used anywhere